index.bs

```diff
@@ -516,6 +522,31 @@ controls the maximum number of attributions for a
 
 # General Algorithms # {#general-algorithms}
 
+<h3 id="parsing-filter-data">Parsing filter data</h3>
+
+To <dfn>parse filter data</dfn> given a |value| and an optional [=boolean=]
```

Comment:
Optional nit: a boolean is slightly less clear than an enum, where the value says what is for.